### DISCUSSION TITLE
Getting this error while trying to use tootctl

### DISCUSSION THREAD

**Discussion by mitexleo** [mastodon/mastodon#20485] [Q&A]
Hi 👋...
I ran out of storage ... cuz , I've added 10+ relays 🙂 .. Actually I didn't know that  Mastodon Caches media from these relays ..... 
Now, I'm trying to remove all cache and after that I'll remove all relays ... (Site is currently not working)

Link : nixorigin.one 

Everytime I run tootctl command I get an error ..

/home/mastodon/live/vendor/bundle/ruby/3.0.0/gems/activerecord-6.1.6/lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client': connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: No such file or directory (ActiveRecord::ConnectionNotEstablished)

How can I fix it ? 

What is causing this problem? ...
TIA.

**Comment by MrGeneration** [comment]
> How can I fix it ?
> 
> What is causing this problem? ...

Your postgresql server is not running but is need to use the mastodon cli.
Expanding storage if possible is the easiest option because if your storage is really full, your pgsql server will refuse to start.
